app/app.py

```diff
@@ -4,144 +4,152 @@
 from flask_restful import Api
 from flask_restful import Resource
 
-app = Flask(__name__)
-api= Api(app)
+APP = Flask(__name__)
+API = Api(APP)
 
-users = {}
-query = []
-answer = []
-answers = {"answer" : answer}
-queries = {"query" : query}
+USERS = {}
+QUERY = []
+ANSWER = []
+ANSWERS = {"ANSWER" : ANSWER}
```

Comment:
This ought to be `ANSWERS = {"answer": answer}`  according to [pep8 whitespaces in expressions and statements](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)